Upgrade speck (#171)
plotly/dash-bio: CHANGELOG.md

# Changelog

## [0.0.4] - 2019-11-02

### Added

> * Added recent update to Speck library to do with jumpy behaviour on click-and-drag.

Reviewer: ```suggestion
* Added recent update to Speck library to fix jumpy behavior on click-and-drag.
```
I personally use Canadian spelling, but all the Plotly docs and materials use American spelling, so...
